components/StoreProducts.js

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import { View } from 'react-native';
+import { ScrollView } from 'react-native-gesture-handler';
+
+import { Button, Title } from '../styles/shared';
+import Product from './Product';
+import StoreCard from './StoreCard';
+
+function filterFruit(product) {
+  if (product) {
+    return product.category.includes('Fruit');
+  }
+  return false;
+}
+
+function filterVegetables(product) {
+  if (product) {
+    return product.category.includes('Vegetables');
+  }
+  return false;
+}
+
+class StoreProducts extends React.Component {
+  detailedStoreTransition = store => {
+    this.props.navigation.navigate('StoresDetailed', {
+      currentStore: store
+    });
+  };
+
+  render() {
+    const { navigation, products, store } = this.props;
+    return (
+      <View>
+        <StoreCard
+          store={store}
+          key={store.id}
+          callBack={() => this.detailedStoreTransition(store)}
+        />
+        {/* TODO @tommypoa can probably make this (e.g Fruits title + fruits products) a sub-component and re-use it in AllProducts */}
```

Comment:
@anniero98 TODO: Good idea if you have time since this pattern is used throughout the app